package.json

```diff
@@ -45,7 +45,7 @@
     "check-tls": "python ./script/tls.py",
     "clang-format": "find atom/ brightray/ chromium_src/ -iname *.h -o -iname *.cc -o -iname *.mm | xargs clang-format -i",
     "lint": "node ./script/lint.js && npm run lint:clang-format && npm run lint:docs",
-    "lint:js": "node ./script/lint.js --js",
+    "lint:js": "node /script/lint.js --js",
```

Comment:
Was the loss of this period intended?

Reply:
Good catch. No, this wasn't intentional. Fixed 282475c86